src/tests/src/solver/modelParser/testModelTranslator.cpp

```diff
@@ -40,10 +40,15 @@ struct Fixture
     Registry<Antares::Solver::Nodes::Node> registry;
```

Comment:
`Registry<Antares::Solver::Nodes::Node> registry;` : do we still need this ?

Reply:
nope